components/eventing-controller/controllers/subscription/beb/reconciler.go

```diff
@@ -92,10 +92,11 @@ func NewReconciler(ctx context.Context, client client.Client, logger *logger.Log
 
 // +kubebuilder:rbac:groups=eventing.kyma-project.io,resources=subscriptions,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=eventing.kyma-project.io,resources=subscriptions/status,verbs=get;update;patch
-// Generate required RBAC to emit kubernetes events in the controller
+// Generate required RBAC to emit kubernetes events in the controller.
 // +kubebuilder:rbac:groups="",resources=events,verbs=create;patch
 // +kubebuilder:rbac:groups=gateway.kyma-project.io,resources=apirules,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:printcolumn:name="Ready",type=bool,JSONPath=`.status.Ready`
```

Comment:
Can you explain why this is not required anymore ? Does it mean the Ready column will get removed ?

Reply:
https://book.kubebuilder.io/reference/markers.html

Reply:
Makes sense! I see it on the go struct here: https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/api/v1alpha1/subscription_types.go#L223